.github/workflows/cd.yml

```diff
@@ -122,7 +122,7 @@ jobs:
 
       - name: Validate input
         run: |
-          [[ "${{ secrets.admin_token }}" ]] || { echo "Set ADMIN_TOKEN variable"; exit 1; }
+          [[ "${{ secrets.tb_admin_token }}" ]] || { echo "Go the tokens section in your workspace and copy the "admin token (user@domain.com)" associated to a user account and set TB_ADMIN_TOKEN as a Secret in your Git repository"; exit 1; }
```

Comment:
could you make sure all text are the same please 🙏  (I guess there are 7 occurrences not updated)

Reply:
Sorry, another example of `NO SABE USARLO` 😅